setup.py

```diff
@@ -0,0 +1,31 @@
+import os
+from glob import glob
+
+from setuptools import setup
+
+PACKAGE_NAME = 'mqtt_ros_bridge'
+
+setup(
+    name=PACKAGE_NAME,
+    version='1.1.0',
+    packages=[PACKAGE_NAME],
+    data_files=[
+        ('share/ament_index/resource_index/packages',
+            ['resource/' + PACKAGE_NAME]),
+        ('share/' + PACKAGE_NAME, ['package.xml']),
+        # Include all launch files.
+        (os.path.join('share', PACKAGE_NAME, 'launch'),
+         glob('launch/*launch.[pxy][yma]*'))
+    ],
+    install_requires=['setuptools'],
+    zip_safe=True,
+    maintainer='benjamin',
+    maintainer_email='bwp18@case.edu',
```

Comment:
Change me to `benjaminwp18` (my git) here

Reply:
Consider adding yourself in this string
https://stackoverflow.com/questions/9999829/how-to-specify-multiple-authors-emails-in-setup-py
Might get whacked by linter tho